File: pset6/mario/less/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

for i in range(height):
    for k in range(height - i - 1):
        print(" ", end="")
    for j in range(i + 1):
        print("#", end="")
    print()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int i = 0; i < height; i++)
    {
        // Create spaces to push the hashes
        for (int k = 0; k < height - i - 1; k++)
        {
            printf(" ");
        }
        // Create the hashes
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // Print a newline
        printf("\n");
    }
}